developer_guide.md

```diff
@@ -10,41 +10,112 @@ To use this website, you need
 
 ## Installation
 
-### 1. Fork the Repository
+### Fork the Repository
```

Comment:
@uma-kt / @priyank-upadhyay  - we can directly clone the repository - isn't it? Why do we want to fork the repository first? 

Is this a recommended practice?

Reply:
Since these steps are for external users, I kept the existing step.
@narmidm / @priyank-upadhyay Please confirm if this step is required.

Reply:
It is required